src/modules/terms/terms.controller.ts

```diff
@@ -0,0 +1,35 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import { ApiBadRequestResponse, ApiBody, ApiCreatedResponse, ApiInternalServerErrorResponse, ApiNotFoundResponse, ApiOperation } from '@nestjs/swagger';
+import { RESPONSE } from '@ukef/constants';
+
+import { CreateFacilityTermRequestItem } from './dto/create-facility-term-request.dto';
+import { TermsService } from './terms.service';
+
+@Controller('terms')
+export class TermsController {
+  constructor(private readonly service: TermsService) {}
+
+  @Post('facilities')
+  @ApiOperation({
+    summary: 'Add a new facility ID to term store.',
+  })
+  @ApiBody({
+    type: CreateFacilityTermRequestItem,
+  })
+  @ApiCreatedResponse({
+    description: RESPONSE.FACILITY_TERM_CREATED,
+    type: CreateFacilityTermRequestItem,
+  })
+  @ApiNotFoundResponse({
+    description: 'The facility ID was not found.',
+  })
+  @ApiBadRequestResponse({
+    description: 'Bad request.',
+  })
+  @ApiInternalServerErrorResponse({
+    description: 'An internal server error has occurred.',
+  })
+  postFacilityToTermStore(@Body() term: CreateFacilityTermRequestItem): Promise<CreateFacilityTermRequestItem> {
```

Comment:
In 3 amigos last week, we decided that we want to make the request body an array containing the object, instead of just the object, to be consistent with the other POST endpoints in TFS and eStore. Sorry, I realise I didn't click "save" on the Jira ticket to update it last week so that part of the spec only appeared just now!

Reply:
Changed request to array.